public/data/ayushlohmod.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Lohmod",
+  "type": "personal",
+  "bio": "I am Ayush Lohmod, currently pursuing Batchelors of Computer Application from SGT UNIVERSITY. I am a tech enthusiast. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/ayushlohmod.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ayushlohmod",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ayushlohmod_",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/ayushlohmod",
```

Comment:
Your LinkedIn profile link doesn't bring any results. _**Please update it with the correct one!**_

![image](https://user-images.githubusercontent.com/88786642/177276631-1a85692c-1a8f-4fcf-9f57-491feb92f307.png)